horizon/pdp.py

```diff
@@ -283,6 +284,16 @@ def _configure_opal_data_updater(self):
             wait_time=1,
         )
 
+    def _configure_opal_offline_mode(self):
+        """
+        configure opal to use offline mode when enabled
+        """
+        opal_client_config.OFFLINE_MODE_ENABLED = sidecar_config.ENABLE_OFFLINE_MODE
+        opal_client_config.STORE_BACKUP_PATH = os.path.join(
+            os.path.dirname(sidecar_config.OFFLINE_MODE_BACKUP_PATH),
+            "policy_store_data.json",
```

Comment:
what about the policy ( rego files ) ?

Reply:
That file has both. I can give it another name if that's confusing